samples/csharp/getting-started/BinaryClassification_CreditCardFraudDetection/CreditCardFraudDetection.Trainer/ModelBuilder.cs

```diff
@@ -47,22 +47,22 @@ public ModelBuilder PreProcessData(MLContext mlContext)
             //Get all the column names for the Features (All except the Label and the StratificationColumn)
             var featureColumnNames = _trainData.Schema.AsQueryable() 
                 .Select(column => column.Name) // Get the column names
-                .Where(name => name != "Label") // Do not include the Label column
+                .Where(name => name != DefaultColumnNames.Label) // Do not include the Label column
```

Comment:
I missed using DefaultColumnNames in some samples while migrating to 0.10.So fixed now.